internal/pkg/service/stream/sink/type/tablesink/keboola/config.go

```diff
@@ -8,10 +8,12 @@ import (
 type Config struct {
 	// EventSendTimeout is a timeout to perform slice upload event or file import event.
 	EventSendTimeout time.Duration `configKey:"eventSendTimeout" configUsage:"Timeout to perform upload send event of slice or import event of file"`
+	JobLimit         int           `configKey:"jobLimit" configUsage:"Number of import jobs running in keboola for single sink"`
 }
 
 func NewConfig() Config {
 	return Config{
 		EventSendTimeout: 30 * time.Second,
+		JobLimit:         2,
```

Comment:
Why 2? :thinking:

Reply:
I think that 1 is too low as there could be issues on cleanup side and it takes some time to cleanup the previous job runs as it is not checked frequently
0332f7eaaaae021286eaace6d1d9ac3a69fecb9c

Reply:
Ah I see. Keboola can accept more jobs but it won't run more than one at a time. So by using 2 here we're allowing to queue one extra one and then we start to throttle. Sounds about right?